nx/lib/nx.ex

```diff
@@ -8714,6 +8714,17 @@ defmodule Nx do
         ]
       >
 
+      iex> u = Nx.tensor([[[1, 1]], [[2, 2]]])
+      iex> v = Nx.tensor([[[1, 2]], [[1, 2]]])
+      iex> Nx.dot(u, [2], [0, 1], v, [2], [0, 1])
+      #Nx.Tensor<
+        s64[2][1]
+        [
+          [3],
+          [6]
+        ]
+      >
```

Comment:
Please add this test to Torchx suite directly instead? The doctests should be about docs mainly. :)

Reply:
I was thinking this way we validate multiple axes work for every backend that runs the doctests, but we can move it to torchx specifically yeah.